File: src/codes/Users_Manual/modulation/csv_1.cs
~using AUTD3Sharp.Modulation;
~using AUTD3Sharp.Modulation.AudioFile;
~using static AUTD3Sharp.Units;
~var path = "path/to/foo.csv";
var m = new Csv(path, 2.0f * kHz, 4u * kHz, new SincInterpolation(new BlackMan(32)));

File: src/codes/Users_Manual/modulation/csv_1.cpp
//~#include<autd3.hpp>
//~#include "autd3/modulation/audio_file/csv.hpp"
//~#include "autd3/modulation/audio_file.hpp"
//~int main() {
//~const auto path = "path/to/foo.csv";
const autd3::modulation::audio_file::Csv m(
    path, 2.0f * autd3::kHz, 4u * autd3::kHz,
    autd3::modulation::SincInterpolation(autd3::modulation::BlackMan{32}));
//~return 0; }